vignettes/detection_efficiency_vignette.Rmd

```diff
@@ -0,0 +1,542 @@
+---
+title: "Estimate Detection Range for 
+           Acoustic Telemetry Receivers"
+author: Benjamin L. Hlina           
+date: "Updated: `r Sys.Date()`"
```

Comment:
I suggest hardcoding the date here to reflect when this document was actually edited. Otherwise, it simply reflects the installation date.

Reply:
I have added the hard coded the date here